dali/operators/decoder/audio/generic_decoder.cc

```diff
@@ -139,6 +140,18 @@ sf_count_t Tell(void *self) {
   return static_cast<MemoryStream *>(self)->Tell();
 }
 
+/**
+ * @brief Produce audio metadata instace from SF_INFO
+ */
+AudioMetadata GetAudioMetadata(SF_INFO &sf_info) {
```

Comment:
How about `const SF_INFO&`?

Reply:
I can try that (I wonder why I didn't mark it const in the first place)